packages/@lwc/babel-plugin-component/vitest.config.mjs

```diff
@@ -0,0 +1,20 @@
+import { mergeConfig, defineProject } from 'vitest/config';
+import sharedConfig from '../../../vitest.shared.mjs';
+import pkg from './package.json';
+
+export default mergeConfig(
+    sharedConfig,
+    defineProject({
+        test: {
+            name: 'babel-plugin-component',
+            // Workaround to fix `const enum`, which is required because we use e.g. `APIFeature` from `@lwc/shared`
```

Comment:
```suggestion
            // To properly resolve `const enum`, we need to point to the TypeScript source files
            // This is required because we use e.g. `APIFeature` from `@lwc/shared`
```
It wasn't immediately clear from the linked issue what the workaround actually is.